pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/SourceEditorController.java

```diff
@@ -126,6 +131,24 @@ public void refreshAST() {
     }
 
 
+    public void showAuxClassPathController(DesignerRoot root) {
+        AuxClassPathController auxClassPathController = new AuxClassPathController(auxclasspathFiles, root);
+
+
+    }
+
+
+    @PersistentProperty
+    public ObservableList<File> getAuxclasspathFiles() {
```

Comment:
The persistence framework throws an exception because you should return a string here, because it can't convert a list that easily. Look at this for example:

https://github.com/pmd/pmd/blob/92e74bb2281ecae65317c525a3b18bd55b546775/pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/MainDesignerController.java#L389-L406